package/boot/uboot-sunxi/patches/252-sunxi-h3-add-support-for-nanopi-r1.patch

```diff
@@ -12,86 +12,54 @@ Signed-off-by: Jayantajit Gogoi <jayanta.gogoi525@gmail.com>
  create mode 100644 arch/arm/dts/sun8i-h3-nanopi-r1.dts
  create mode 100644 configs/nanopi_r1_defconfig
 
---- a/arch/arm/dts/Makefile
-+++ b/arch/arm/dts/Makefile
-@@ -625,6 +625,7 @@ dtb-$(CONFIG_MACH_SUN8I_H3) += \
- 	sun8i-h3-nanopi-m1-plus.dtb \
- 	sun8i-h3-nanopi-neo.dtb \
- 	sun8i-h3-nanopi-neo-air.dtb \
-+	sun8i-h3-nanopi-r1.dtb \
- 	sun8i-h3-orangepi-2.dtb \
- 	sun8i-h3-orangepi-lite.dtb \
- 	sun8i-h3-orangepi-one.dtb \
---- /dev/null
+--- a/arch/arm/dts/sun8i-h3-nanopi-r1.dts
 +++ b/arch/arm/dts/sun8i-h3-nanopi-r1.dts
```

Comment:
Is this patch needed at all any more? It looks like support for FriendlyARM NanoPi R1 was added upstream, our version is probably even older.

Reply:
U-Boot imported the dts file from Linux kernel in u-boot/u-boot@2c597855aa17, but the defconfig for NanoPi R1 has not been added, so I left it in our patch.

And I first thought some nodes in the dts were overriden intentionally for our purpose (like reusing power button for restart), but I've found that the same dts was added for linux in e6d9f6fdffc6 and superseded by the upstreamed version, so now I think we can just drop the rest of the dts patch.